x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -427,4 +438,8 @@ export function registerEncryptedSavedObjects(
     ]),
   });
   // Encrypted saved objects
+  encryptedSavedObjects.registerType({
+    type: MESSAGE_SIGNING_KEYS_SAVED_OBJECT_TYPE,
+    attributesToEncrypt: new Set(['private_key', 'public_key', 'passphrase']),
```

Comment:
do we want to encrypt the public_key or should we just leave it unencrypted?

Reply:
I would say yes, but let's see what @aleksmaus thinks about it

Reply:
yeah, public key is public